guide/src/free-threading.md

```diff
@@ -11,9 +11,11 @@ If you want more background on free-threaded Python in general, see the [what's
 new](https://docs.python.org/3.13/whatsnew/3.13.html#whatsnew313-free-threaded-cpython)
 entry in the CPython docs, the [HOWTO
 guide](https://docs.python.org/3.13/howto/free-threading-extensions.html#freethreading-extensions-howto)
-for porting C extensions, and [PEP 703](https://peps.python.org/pep-0703/),
-which provides the technical background for the free-threading implementation in
-CPython.
+for porting C extensions the [extension porting
+guide](https://py-free-threading.github.io/porting-extensions/) in the
```

Comment:
This sentence looks like it might need some punctuation now? 🤔 

'... the HOWTO guide for porting C extensions the extension porting guide in the ...'